packages/openassistants/openassistants/llm_function_calling/infilling.py

```diff
@@ -67,16 +68,37 @@ async def generate_argument_decisions(
     ]
 
     result = await generate_to_json(
-        chat, final_messages, json_schema, "generate_argument_decisions"
+        chat,
+        final_messages,
+        json_schema,
+        "generate_argument_decisions",
+        tags=["generate_argument_decisions"],
     )
 
     return result
 
 
+def entity_to_json_schema_obj(entity: Entity):
+    d = {"const": entity.id}
+    if entity.metadata is not None:
+        d["$comment"] = entity.metadata
```

Comment:
https://github.com/OAI/OpenAPI-Specification/issues/348#issuecomment-336194030

I think we should use 'description' here